clients/constants.go

```diff
@@ -24,36 +24,6 @@ const (
 	CorrelationHeader    = "X-Correlation-ID" // Sets the key of the Correlation ID HTTP header
 )
 
-// Constants related to defined routes in the service APIs
```

Comment:
Does it make sense to move the remaining constants to the common constants? i.e. these remaining are not specific to using clients.

Reply:
If agree to move these, please use separate commit so reviewing those changes can be done with out looking thru all the previous changes... ;-)

Reply:
Agree! Moved the remaining constants to the common package.